server/src/language-service/opctagListProvider.ts

```diff
@@ -0,0 +1,42 @@
+import { SepticReferenceProvider } from "../septic";
+
+export class OpcTagListProvider {
+    constructor() {}
+
+    public generateOpcTagList(refProvider: SepticReferenceProvider): string {
+        return "";
+    }
+}
+
+export function generateOpcReport(
+    refProvider: SepticReferenceProvider
+): string {
+    let header: string = "ObjectId;ObjectType;ObjectAttribute;OPCTag;";
+    let entries: string[] = [];
+    let opcObjects = refProvider.getObjectsByType(
+        "SopcTvr",
```

Comment:
You may want to check with @mortenfre whether there are any other Sopc* objects that should be included in the dump (SopcSampleTvr, SopcChangeEvr, SopcAsyncEvr, SopcTimeTvr, SopcTextTvr, SopcSvr)

Reply:
Are any of these in use?